OptimizelySDK/Odp/OdpEventApiManager.cs

```diff
@@ -127,21 +131,21 @@ public bool SendEvents(string apiKey, string apiHost, List<OdpEvent> events)
             string data
         )
         {
-            var request = BuildRequestMessage(apiKey, endpoint, data);
+            var request = BuildOdpEventMessage(apiKey, endpoint, data);
 
-            var response = await _httpClient.SendAsync(request);
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
Why do we need override here? Each client may set their own TLS version like 1.3.

Reply:
I see. Wondering if we have the same issue with other servers too (datafile, event, and odpEvent) and if we can set "minimum" version to 1.2 instead of fixing the version.